### PR TITLE
Added Terms

### PR DESCRIPTION
**Pull request by Miya25**
Added Terms And Services To UniverseList Source Code.

### CODE REVIEW

File: src/pages/legal/terms.ejs [hunk 1]
```diff
@@ -4,15 +4,74 @@
...
+        <p>
+          From the moment you post something on the website, we save your profile picture, user ID, username, and discriminator.
+        </p>
+        <p>
+          B. How long do we retain your data?
+        </p>
+        <p>
+          We keep your data when you have posted content on our website. When you don't have any content or delete them, our programmed system automatically deletes everything. This is not a cause for concern because the information obtained is obtained for security purposes as well as for a future scheduled backup.
+        </p>
+       <p>C. Location Information</p>
```
Comment (TejasLamba2006) [inline src/pages/legal/terms.ejs:49]: Universe List doesnt store IP data or geolocation data

**Review by TejasLamba2006 — CHANGES_REQUESTED** [review]
Kindly changes some part as Universe List doesnt store geolocation data until date.

File: src/pages/legal/terms.ejs (hunk 1 above)
Comment (Miya25) [reply src/pages/legal/terms.ejs:49]: Sorry about that,I'll fix it later

**Comment by ishaantek** [comment]
Thank you for submitting the pull request. I've reviewed the changes, and overall, it looks good. I appreciate your effort in adding the TOS to our project.

I'll make a few minor changes to the text. Once these changes are made, I will proceed with merging your pull request into the main branch.

Thank you again for your contribution!

**Comment by Miya25** [comment]
:3 -RS

**Comment by Miya25** [comment]
i'll now create a PR for prettifying the code

**Comment by TejasLamba2006** [comment]
What's you discord so we can give you a sweet role?

**Comment by Miya25** [comment]
I'm not willing to share that.

**Comment by TejasLamba2006** [comment]
It was in your github profile
`ranveersoni#1894`

**Comment by Miya25** [comment]
mhm?

**Comment by Miya25** [comment]
Any updates  here?

**Comment by TejasLamba2006** [comment]
It would take some time as @ishaantek isn't that much active now because of exams.

I would merge it if I had the power's but I don't sadly

**Comment by Miya25** [comment]
mhm.

**Pull request merged**